src/Engines/thinEngine.ts

```diff
@@ -722,15 +724,35 @@ export class ThinEngine {
 
             // Exceptions
             if (navigator && navigator.userAgent) {
-                let ua = navigator.userAgent;
+                // Function to check if running on mobile device
+                this._checkForMobile = () => {

```

Comment:
Reminds me why we need that isMobile check again?

Reply:
So I made the userAgent sniffing leaner after testing again on my iPhone.  I still need the 'Mac'/'ontouchend' check because 'Mobile' isn't in the userAgent when run on an iPad Pro with the latest version of iOS. @deltakosh @sebavan lemme know what you think.

Reply:
As far as I know, isMobile is used with FreeCameraTouchInput to potentially treat non-mobile touchscreen inputs as touch events, rather than default them to mouse events.  It's also used in PointerDragBehavior for some specific mouse handling logic, I think.